libraries/Microsoft.Bot.Connector.Client/Bot.Builder/BotFrameworkClient.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Schema;
+
+namespace Microsoft.Bot.Connector.Client.Bot.Builder
```

Comment:
The original files in this folder Bot.Builder kept the namespace "Microsoft.Bot.Builder". We should either maintain the namespace exception (which reduces the updates needed in referencing libraries), or eliminate this folder and move the files one level up with the proper namespace.